libraries/AP_TECS/AP_TECS.cpp

```diff
@@ -1335,7 +1336,7 @@ void AP_TECS::update_pitch_throttle(int32_t hgt_dem_cm,
     }
 
     if (flight_stage == AP_FixedWing::FlightStage::TAKEOFF || flight_stage == AP_FixedWing::FlightStage::ABORT_LANDING) {
-        if (!_flags.reached_speed_takeoff && _TAS_state >= _TAS_dem_adj) {
+        if (!_flags.reached_speed_takeoff && _TAS_state >= _TAS_dem && (_TAS_dem > 3.0f)) {
```

Comment:
i think we could make this > 0 not > 3, the > 3 we used during testing was paranoia

Reply:
I did at first, but it didn't work.
In the catapult launch, it seems that TECS is initialized after the aircraft is in the air.
I'll give it another shot, just in case I misrememeber.

Reply:
maybe use _TAS_dem > min airspeed then?

Reply:
That actually sounds much better!